lib/Slider/Slider.js

```diff
@@ -579,6 +586,10 @@ module.exports = kind(
 			this.updatePopup(this.elasticFrom);
 			this.set('value',this.elasticFrom);
 
+			if (this.enableJumpIncrement) {
+				this.updateButtonStatus(v);
```

Comment:
Hi @aarontam, it's good feedback considering performance. But before moving this to ```dragfinish```, I want to get UX team's feedback if there is no issue although position is min/max and user is still draging (not dragfinish)